### DISCUSSION TITLE
failed to download archive from http://source-controller.flux-system.svc.cluster.local./gitrepository/flux-system/openshift-repo/abc.tar.gz407 Proxy Authentication Required

### DISCUSSION THREAD

**Discussion by nurubhas** [fluxcd/flux2#3447] [General]
I have issue with my FluxCD setup.. I have deployed FluxCD as a operator in Openshift 4.9 and I have added a GitRepository in FluxCD... I could able to see all reconcilation was happening based on application yaml manifest.

So here authentication was succeed from Flux to gitops repo.

I just initiated Kustomization to deploy app resources in namespace.. observed below error message
~~~
Generated from kustomize-controller
13 times in the last 1 hour
failed to download archive from http://source-controller.flux-system.svc.cluster.local./gitrepository/flux-system/openshift-repo/251f85a4b6bcb1b96e46777e5ff32bfb76dfcb25.tar.gz, status: 407 Proxy Authentication Required
~~~

~~~
apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
kind: Kustomization
metadata:
  name: simpleapp
  namespace: flux-system
spec:
  force: false
  interval: 5m
  path: ./simple-app
  prune: true
  sourceRef:
    kind: GitRepository
    name: openshift-repo
  targetNamespace: argocd    # even i used flux-system namespace observed same error message
  timeout: 2m
  wait: true
~~~

Can you guys help... anything missing ?

**Comment by tahirraza** [comment]
You mentioned code was successfully pulled. 
Check that your git repo READY/STATUS output `oc get gitrepository -A` 

Other relevant things you can share:
- What version of Flux
- How did you bootstrap (share command if did via cli, redact anything sensitive)

**Reply by johnlabarge** [reply]
Same issue here. Git repo shows ready status.
